### PR TITLE
Add integration tests to yarn test, and remove src maps in test

### PR DESCRIPTION
**Pull request by jonsherrard**
Add integration tests to yarn test, and remove src maps in test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "PORT=4321 NODE_ENV=test LOG_LEVEL=silent mocha ./src/**/*.test.js ./test/server/*.test.js --require test/babel-register --bail",
-    "test:integration": "NODE_ENV=test mocha ./test/integration/**/*.test.js --bail"
+    "test": "PORT=4321 NODE_ENV=test LOG_LEVEL=silent mocha ./src/**/*.test.js ./test/**/*.test.js --require test/babel-register --bail"
```
Comment (thomasdigby) [inline package.json]: Add `--timeout 5000` here to save the duplication

File: src/config/createWebpackConfig.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const mainBabelOptions = {
 
 module.exports = () => {
   let config = {
-    devtool: 'cheap-module-source-map',
+    devtool: (process.env.NODE_ENV == 'test') ? false : 'cheap-module-source-map',
```
Comment (thomasdigby) [inline src/config/createWebpackConfig.js]: 🙌

File: package.json (hunk 1 above)
Comment (jonsherrard) [reply package.json]: I don't want all the tests to have a timeout of 5000, would be good to realise if rendering a component goes over 2 seconds for example.

Comment (jonsherrard) [reply package.json]: Although if that does happen I would probably give up.

Comment (thomasdigby) [reply package.json]: Ah yes, I only added it to the integration tests.

**Pull request merged**